### PR TITLE
Created db classes

### PR DESCRIPTION
**Pull request by pstowski**

### CODE REVIEW

File: src/main/java/pl/polsl/bd/clinicassistance/database/Address.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package pl.polsl.bd.clinicassistance.database;
+
+        import lombok.Getter;
```
Comment (Reivals) [inline src/main/java/pl/polsl/bd/clinicassistance/database/Address.java]: Proper format for imports

File: src/main/java/pl/polsl/bd/clinicassistance/database/Address.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+@Entity
+public class Address {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Setter
+    @Getter
```
Comment (Reivals) [inline src/main/java/pl/polsl/bd/clinicassistance/database/Address.java]: Why dont you just add @Setter and @Getter annotation at class level?

File: src/main/java/pl/polsl/bd/clinicassistance/database/AppointmentStatus.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package pl.polsl.bd.clinicassistance.database;
+
+public enum AppointmentStatus {
```
Comment (Reivals) [inline src/main/java/pl/polsl/bd/clinicassistance/database/AppointmentStatus.java]: Do we need seperated instance of that? Cant we declare it in Appointment class?

File: src/main/java/pl/polsl/bd/clinicassistance/database/EmployeeRole.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package pl.polsl.bd.clinicassistance.database;
+
+public enum EmployeeRole {
```
Comment (Reivals) [inline src/main/java/pl/polsl/bd/clinicassistance/database/EmployeeRole.java]: Do we need seperate enum? I recon this enum will be only used in Employee class

File: src/main/java/pl/polsl/bd/clinicassistance/database/Examination.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+@Entity
+@Table(name = "examination_type")//index on code is redundant, because it is primary key.
+public class Examination {
+
+    @Setter
+    @Getter
+    @Id
+    @Column(length = 4)
+    private String code;
```
Comment (Reivals) [inline src/main/java/pl/polsl/bd/clinicassistance/database/Examination.java]: You dont guarantee uniqueness of this field -> either create PK or add to this field index + unique constraint

File: src/main/java/pl/polsl/bd/clinicassistance/database/LaboratoryExaminationStatus.java [hunk 6]
```diff
@@ -0,0 +1,9 @@
+package pl.polsl.bd.clinicassistance.database;
+
+public enum LaboratoryExaminationStatus {
```
Comment (Reivals) [inline src/main/java/pl/polsl/bd/clinicassistance/database/LaboratoryExaminationStatus.java]: Do we need seperated enum ?

File: src/main/java/pl/polsl/bd/clinicassistance/database/ExaminationDomain.java [hunk 7]
```diff
@@ -0,0 +1,11 @@
+package pl.polsl.bd.clinicassistance.database;
+
+public enum ExaminationDomain {
```
Comment (Reivals) [inline src/main/java/pl/polsl/bd/clinicassistance/database/ExaminationDomain.java]: Do we need seperated enum ?

File: src/main/resources/hibernate.cfg.xml [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+        <property name="dialect">org.hibernate.dialect.PostgreSQL95Dialect</property>
+
+        <!-- Echo the SQL to stdout -->
+        <property name="show_sql">true</property>
+
+        <!-- Set the current session context -->
+        <property name="current_session_context_class">thread</property>
+
+        <!-- Drop and re-create the database schema on startup -->
+        <property name="hbm2ddl.auto">create-drop</property>
```
Comment (Reivals) [inline src/main/resources/hibernate.cfg.xml]: should it be create-drop?

File: src/main/java/pl/polsl/bd/clinicassistance/database/Employee.java [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Setter
+    @Getter
+    @Column(nullable = false)
+    private String document;
+
+    @Setter
+    @Getter
+    @Column
+    private String eligibility;
```
Comment (Reivals) [inline src/main/java/pl/polsl/bd/clinicassistance/database/Employee.java]: What is it?

Comment (Reivals) [reply src/main/java/pl/polsl/bd/clinicassistance/database/Employee.java]: eligibility for what :P

File: src/main/java/pl/polsl/bd/clinicassistance/database/Address.java (hunk 1 above)
Comment (pstowski) [reply src/main/java/pl/polsl/bd/clinicassistance/database/Address.java]: fixed

File: src/main/java/pl/polsl/bd/clinicassistance/database/Address.java (hunk 2 above)
Comment (pstowski) [reply src/main/java/pl/polsl/bd/clinicassistance/database/Address.java]: fixed

File: src/main/java/pl/polsl/bd/clinicassistance/database/AppointmentStatus.java (hunk 3 above)
Comment (pstowski) [reply src/main/java/pl/polsl/bd/clinicassistance/database/AppointmentStatus.java]: fixed

File: src/main/java/pl/polsl/bd/clinicassistance/database/EmployeeRole.java (hunk 4 above)
Comment (pstowski) [reply src/main/java/pl/polsl/bd/clinicassistance/database/EmployeeRole.java]: fixed

File: src/main/java/pl/polsl/bd/clinicassistance/database/LaboratoryExaminationStatus.java (hunk 6 above)
Comment (pstowski) [reply src/main/java/pl/polsl/bd/clinicassistance/database/LaboratoryExaminationStatus.java]: fixed

File: src/main/java/pl/polsl/bd/clinicassistance/database/ExaminationDomain.java (hunk 7 above)
Comment (pstowski) [reply src/main/java/pl/polsl/bd/clinicassistance/database/ExaminationDomain.java]: fixed

File: src/main/resources/hibernate.cfg.xml (hunk 8 above)
Comment (pstowski) [reply src/main/resources/hibernate.cfg.xml]: At this stage it should be create-drop

File: src/main/java/pl/polsl/bd/clinicassistance/database/Employee.java (hunk 9 above)
Comment (pstowski) [reply src/main/java/pl/polsl/bd/clinicassistance/database/Employee.java]: e.x. eligibility for practice medice, eligibility for work at laboratory

**Pull request merged**